src/api/app/models/user.rb

```diff
@@ -675,6 +675,30 @@ def owned_packages
     owned
   end
 
+  def involved_items_as_owner(filters = {})
+    items = []
+
+    OwnerSearch::Owned.new.for(self).each do |owner|
```

Comment:
This crashes if you don't have the attribute

![Screenshot from 2020-10-16 13-26-51](https://user-images.githubusercontent.com/514785/96253100-49674780-0fb3-11eb-8612-154d48e2bb7e.png)

Reply:
That [is just missing configuration, no?](https://github.com/openSUSE/open-build-service/issues/4861#issuecomment-417377553)

Reply:
Nope. You can set this attribute on some project or you can not.

Reply:
Then let's not display an option to filter by "Owner" in the roles filter, when there isn't any project with the `OBS:OwnerRootProject` attribute set. On it.

Reply:
It should be fixed now. Please have a look again.